core/src/main/java/cucumber/runtime/RuntimeOptionsFactory.java

```diff
@@ -86,7 +86,7 @@ private void addPlugins(CucumberOptions options, List<String> args) {
         for (String plugin : plugins) {
             args.add("--plugin");
             args.add(plugin);
-            if (PluginFactory.isFormatterName(plugin)) {
+            if (PluginFactory.isPluginName(plugin)) {
                 pluginSpecified = true;
```

Comment:
Will try :)

Reply:
Removing the if statement, changes the result of `RuntimeOptionsFactoryTest.create_null_formatter_when_no_formatter_plugin_is_defined()` so that "cucumber.runtime.formatter.AnyStepDefinitionReporter" plugin is used, rather than "cucumber.runtime.formatter.NullFormatter".

Reply:
Okay. So now the null formatter is gone (replaced by no formatter), this test should be changed to check if no formatter is there instead.